### PR TITLE
Fix ``useless-suppression`` for ``wrong-import-order``

### LINKED ISSUE #2366: useless-suppression false positive with ungrouped-imports
### Steps to reproduce

Run pylint on the following code:

```python
# pylint: enable=useless-suppression
from pylint import run_pylint
import astroid
from pylint import run_pyreverse # pylint: disable=ungrouped-imports
```

### Current behavior

This message is issued:
```
4:0: I0021: Useless suppression of 'ungrouped-imports' (useless-suppression)
```
However, if the comment is removed from line 4, this message is issued:
```
4:0: C0412: Imports from package pylint are not grouped (ungrouped-imports)
```

### Expected behavior

Since there is an actual message being suppressed, I would expect to get no `useless-suppression` message.

### pylint --version output

```
pylint 2.0.1
astroid 2.0.1
Python 3.4.6 (default, Mar 22 2017, 12:26:13) [GCC]
```

### PR DESCRIPTION
**Pull request by DanielNoord**
- [x] Add yourself to CONTRIBUTORS if you are a new contributor.
- [x] Add a ChangeLog entry describing what your PR does.
- [x] If it's a new feature, or an important bug fix, add a What's New entry in
      `doc/whatsnew/<current release.rst>`.
- [x] Write a good description on what the PR does.

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :bug: Bug fix          |

## Description

This also adds a new method to ``MessagesHandlerMixIn`` which adds messages to the list of the ignored messages without doing anything else. 
This will be useful for some other ``useless-suppression`` false positives.
This closes #2366

### CODE REVIEW

File: pylint/message/message_handler_mix_in.py [hunk 1]
```diff
@@ -257,6 +257,33 @@ def add_message(  # type: ignore # MessagesHandlerMixIn is always mixed with PyL
...
+        """Prepares a message to be added to the ignored message storage
+
+        When earlier checks make it so add_message() is never called
+        useless-suppression will create false-positives.
+        This function avoids this by adding those message to the ignored msgs attribute
+        """
+        message_definitions = self.msgs_store.get_message_definitions(msgid)
+        for message_definition in message_definitions:
+            self.check_message_definition(message_definition, line, node)
+            self.file_state.handle_ignored_message(
```
Comment (DanielNoord) [inline pylint/message/message_handler_mix_in.py:276]: I think I'm going to make another PR with a refactor of `FileState.handle_ignored_message()`. There is no need to pass `node`, `args` and `confidence` here and I don't think there is a valid reason for keeping the parameters in there.
This will in return also make the arguments needed for `add_ignored_message` more intuitive.

**Review by Pierre-Sassoulas — APPROVED** [review]
Nice fix !
